pylint/test/functional/consider_merging_isinstance.txt

```diff
@@ -0,0 +1,5 @@
+consider-merging-isinstance:15:isinstances:Consider merging these isinstance calls to var[3].
```

Comment:
Maybe we should try to print how should it look exactly?
So in the case of `` if isinstance(var[3], int) or isinstance(var[3], float) or isinstance(var[3], list)`` it can look as:

    $ pylint a.py
    1:R:Consider merging these calls to:
         isinstance(var[3], (int, float, list))